aea/cli/transfer.py

```diff
@@ -0,0 +1,109 @@
+# -*- coding: utf-8 -*-
+# ------------------------------------------------------------------------------
+#
+#   Copyright 2018-2020 Fetch.AI Limited
+#
+#   Licensed under the Apache License, Version 2.0 (the "License");
+#   you may not use this file except in compliance with the License.
+#   You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#   Unless required by applicable law or agreed to in writing, software
+#   distributed under the License is distributed on an "AS IS" BASIS,
+#   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#   See the License for the specific language governing permissions and
+#   limitations under the License.
+#
+# ------------------------------------------------------------------------------
+from aea.common import Address
+from aea.configurations.constants import DEFAULT_LEDGER
+from aea.crypto.ledger_apis import LedgerApis
+import time
+
+
+"""Implementation of the 'aea get_wealth' subcommand."""
+
+from typing import Dict, Optional, cast
+
+import click
+
+from aea.cli.utils.context import Context
+from aea.cli.utils.decorators import check_aea_project
+from aea.cli.utils.package_utils import (
+    get_wallet_from_context,
+    try_get_balance,
+    verify_or_create_private_keys_ctx,
+)
+from aea.crypto.registries import ledger_apis_registry
+from aea.crypto.wallet import Wallet
+DEFAULT_SETTLE_TIMEOUT = 60
+
+
+@click.command()
+@click.argument(
+    "type_",
+    metavar="TYPE",
+    type=click.Choice(ledger_apis_registry.supported_ids),
+    #default=DEFAULT_LEDGER,
+    required=True,
+)
+@click.argument(
+    "address", type=str, required=True,
+)
+@click.argument(
+    "amount", type=int, required=True,
+)
+@click.argument(
+    "fee", type=int, required=False, default=100
+)
+@click.option(
+    "-y", "--yes", type=bool, is_flag=True, default=False
+)
+@click.option(
+    "--settle-timeout", type=int, default=DEFAULT_SETTLE_TIMEOUT
+)
+@click.pass_context
+@check_aea_project
+def transfer(click_context, type_, address, amount, fee, yes, settle_timeout):
+    """Get the wealth associated with the private key."""
+    ctx = cast(Context, click_context.obj)
+    if not yes:
+        click.confirm(
+            f'Transfer from {type_} wallet to {address}, amount {amount} with fee {fee}. Do you want to continue?', 
+            abort=True
+        )
+    tx_digest = do_transfer(ctx, type_, address, amount, fee)
+    if not tx_digest:
+        raise click.ClickException(f'failed to send!: {tx_digest}')
```

Comment:
if there is no tx_digest, makes no sense to print it

Reply:
done